daemon/delete_unix.go

```diff
@@ -0,0 +1,35 @@
+// +build !windows
+
+package daemon
+
+import (
+	"fmt"
+
+	"github.com/docker/docker/pkg/stringid"
+)
+
+// VolumeRm removes the volume with the given name.
+// If the volume is referenced by a container it is not removed
+// This is called directly from the remote API
+func (daemon *Daemon) VolumeRm(name string) error {
+	v, err := daemon.volumes.Get(name)
+	if err != nil {
+		return err
+	}
+	if err := daemon.volumes.Remove(v); err != nil {
+		if err == ErrVolumeInUse {
+			containers := []string{}
+			for _, container := range daemon.List() {
```

Comment:
It's not currently stored, but could be. On `daemon.Volumes.Increment` we can pass in the container and have it store those refs (would be cool to show this on inspect too).